src/Widgets/AlbumImage.vala

```diff
@@ -28,10 +27,11 @@ public class Music.AlbumImage : Gtk.Grid {
             pixel_size = 200
         };
 
+        css_classes = {"card"};
         halign = Gtk.Align.CENTER;
         valign = Gtk.Align.CENTER;
         height_request = 200;
         width_request = 200;
-        add (image);
+        attach (image, 0, 0);
```

Comment:
`Gtk.Box`, `append()`? Unless you actually have multiple columns and rows here - but I've seen elementary apps using a lot of unnecessary grids so saying just in case.

Reply:
Yeah it used to be that they were saying Box was going to be removed in GTK4 so we were avoiding it. Box probably works here

Reply:
Nah, that was never the case. HBox and VBox were - and they _were_ removed, GtkBox was new in GTK3.

There were a few misleading statements in docs though, but those had been corrected since.